staging/src/slime.io/slime/modules/lazyload/charts/templates/cluster-global-sidecar.yaml

```diff
@@ -290,11 +313,7 @@ spec:
           typed_per_filter_config:
             envoy.filters.http.lua:
               "@type": type.googleapis.com/envoy.extensions.filters.http.lua.v3.LuaPerRoute
-              source_code:
```

Comment:
why we move the add-source-ns-header lua code from per-route filter config to filter config?

Reply:
compatiable with istio 1.7, cause istio 1.7 related envoy does support source_code param